components/director/internal/domain/bundle/resolver.go

```diff
@@ -235,6 +237,16 @@ func (r *Resolver) DeleteBundle(ctx context.Context, id string) (*graphql.Bundle
 		return nil, err
 	}
 
+	err = r.apiSvc.DeleteAllByBundleID(ctx, id)
```

Comment:
Just an idea - shouldn't this be part of the bundle service and not the resolver? My idea behind this is that the business logic (service) should contain the deletion of relative resources.
Not sure if this was already discussed.

Reply:
As discussed this is accepted by design